ZenPacks/zenoss/Microsoft/Windows/WinService.py

```diff
@@ -109,6 +109,74 @@ def monitored(self):
                     if self.getMonitored(datasource):
                         return True
 
+        # 3 check the service class
+        if hasattr(self, 'serviceclass') and 'serviceclass' in self.getRelationshipNames():
```

Comment:
When the Microsoft.Windows ZenPack was made, it was decided to move the monitoring of services to monitoring templates instead of using the "serviceclass" or Infrastructure -> Windows Services page.  We are now enabling the page so that services can be organized together for viewing status.  This will also allow the user to enable monitoring from that page.  It is a similar thing to ip services and os processes.